langserver/util.go

```diff
@@ -35,6 +35,21 @@ func IsVendorDir(dir string) bool {
 	return strings.HasPrefix(dir, "vendor/") || strings.Contains(dir, "/vendor/")
 }
 
+// isUri tells if s denotes an URI
+func isUri(s string) bool {
+	return strings.HasPrefix(s, "file:///")
+}
+
+// pathToUri converts given path to file URI
+func pathToUri(path string) string {
```

Comment:
Can you call this `pathToURI`. Can we also document that path has to be absolute?